### ISSUE TITLE
Any method to check existence of an element with a wait and that returns a boolean

### ISSUE THREAD

**Issue by Srikanth-Mondem** [cheezy/page-object#389]
I see methods like wait_until, when_present are throwing timeout exceptions if element is not visible. But in some cases some elements might be random and if that's not present also my test should continue. Can we get any method like that?

Methods like name_element.visible? or .present? will do return boolean but they wont wait unless we give hard sleep.

I can use begin rescue to catch exception and proceed, but that's going to be too many lines and looks messy.

In short, Looking for a method to do something like
{
sleep 5
name_element.visible?
}

**Comment by cheezy** [comment]
@Srikanth-Mondem sounds like you are asking for a method that works like this:

`name_element.click if name_element.**check_visible(5)**`

The `check_visible` method would not throw an exception but would simply return true or false depending on if the element was visible within the time provided.

**Comment by Srikanth-Mondem** [comment]
Yes, that's the one I am looking for!!

Thanks @cheezy
